packages/@ember/application/lib/application.js

```diff
@@ -483,13 +483,22 @@ const Application = Engine.extend({
     @method waitForDOMReady
   */
   waitForDOMReady() {
-    if (!this.$ || this.$.isReady) {
+    if (!hasDOM || this.documentReadyState() !== 'loading') {
       schedule('actions', this, 'domReady');
     } else {
-      this.$().ready(bind(this, 'domReady'));
+      let callback = function() {
```

Comment:
Use an arrow function here, then you don't need `bind(...)`. Otherwise, `bind(...)` will return a different function than the variable in `callback`, and your removal code won't find it.

Reply:
Whoops! Already fixed it